<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFormatDlg</class>
 <widget class="QDialog" name="CFormatDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>分区格式化窗口</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbStatus">
     <property name="text">
      <string>请选择隐藏分区大小</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlSizeSelect">
     <item>
      <widget class="QSlider" name="hsSize">
       <property name="minimum">
        <number>4</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbSize">
       <property name="text">
        <string>10 G</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbConfirm">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hsSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CFormatDlg</receiver>
   <slot>sltValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbConfirm</sender>
   <signal>clicked()</signal>
   <receiver>CFormatDlg</receiver>
   <slot>sltConfirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>CFormatDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sltValueChanged(int)</slot>
  <slot>sltConfirm()</slot>
 </slots>
</ui>
